### PR TITLE
Added Remove duplicate script and updated requirements.txt

### PR DESCRIPTION
**Pull request by SKAUL05**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -14,3 +14,4 @@ scikit-fuzzy
 sklearn
 sympy
 tensorflow
+ordered_set
```
Comment (cclauss) [inline requirements.txt]: Looks like it breaks the alphabetical ordering on requirements.txt.

Why do we need to add this module?

File: strings/remove_duplicates.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    >>> remove_duplicates("Python is great and Java is also great")
+    'Python is great and Java also'
+    """
+    sen_list = sentence.split(" ")
+    check = OrderedSet()
+
+    for a_word in sen_list:
+        check.add(a_word)
+
+    return " ".join(check)
```
Comment (cclauss) [inline strings/remove_duplicates.py]: return " ".join(sorted(check))  # and lose the dependency.

Comment (SKAUL05) [reply strings/remove_duplicates.py]: That seems correct

**Comment by cclauss** [comment]
git rm strings/check_panagram.py~HEAD

**Pull request closed without merging**